### ISSUE TITLE
Find out the best way to write code for both Node JS and the browser

### ISSUE THREAD

**Issue by bjornharrtell** [bjornharrtell/javascript.util#9]
I've looked into Require JS but haven't yet decided because I'm not sure how to use it. Haven't seen any other solution though.

**Comment by w8r** [comment]
Beginning from 0.15.0 you cannot install jsts as an npm module to use with browserify.
```
Uncaught TypeError: fs.readFileSync is not a function
```
Systems: Mac OS X 10.10.2, Win 8.1
It's caused by bower build in javascript.utils and both CommonJS `require`s and `goog.require` I guess. Duplicating the issue here to get more attention.

**Comment by bjornharrtell** [comment]
Unfortunately I think there are additional issues. Plainly running npm install jsts or npm install in a github clone works but when including jsts as dependency via package.json I get this error when running npm install:

```
> grunt

>> Local Npm module "grunt-bower-task" not found. Is it installed?
>> Local Npm module "grunt-closure-tools" not found. Is it installed?
>> Local Npm module "grunt-gjslint" not found. Is it installed?
>> Local Npm module "grunt-contrib-concat" not found. Is it installed?
>> Local Npm module "grunt-gh-pages" not found. Is it installed?
Warning: Task "bower" not found. Use --force to continue.

It seems needed dependencies of javascript.util is not installed in this case. Using a built version of javascript.util for Node Js would reduce the complexity of npm install (now it does alot of work so I think perhaps I'll go for that to fix this.

**Comment by bjornharrtell** [comment]
So it seems using the built version resolves issues for both usage with Node JS and Browserify. Released as 0.12.12 which should mean it will be used with JSTS automatically. Reopen if I'm mistaken.

**Comment by w8r** [comment]
Thanks!
